packages/manager/src/services/rbac/queries.tsx

```diff
@@ -0,0 +1,117 @@
+/* eslint-disable */
+import { useMutation, useQuery, useQueryClient } from '@tanstack/react-query';
+import { orchestratorReq } from '@app/config/orchestratorReq';
+import {
+  TCreateApiKeyRes,
+  TIndividualRole,
+  TUserList,
+  TRoverGroupList,
+  TMemberforSPA,
+  TAddPermissionDTO,
+  TDeletePermissionDTO,
+  TRoleforMember
+} from './types';
+import toast from 'react-hot-toast';
+const rbacKeyQueryKeys = {
+  list: (webPropertyIdentifier: string) => ['rbac-keys', webPropertyIdentifier] as const
+};
+
+// fetches userlist from rover from name string
+export const fetchUserlist = async (name: string): Promise<TUserList[]> => {
+  const { data } = await orchestratorReq.get(`/sot/rover/user/${name}`);
+  return data.data;
+};
+
+// fetches rover group from rover
+export const fetchRoverGroup = async (group_name: string): Promise<TRoverGroupList[]> => {
```

Comment:
Hi @nikhitamore , we are following camel casing in the project, could you please the _ separated as camel case?